hack/logging-dump.sh

```diff
@@ -122,105 +117,107 @@ check_project_info() {
 get_env() {
   local pod=$1
   local env_file=$2/$pod
-  containers=$(oc get po $pod -o jsonpath='{.spec.containers[*].name}')
+  local ns=${3:-$NAMESPACE}
+  local pattern=${4:-"Dockerfile-*logging*"}
+  echo ---- Env for $pod
+  containers=$(oc -n $ns get po $pod -o jsonpath='{.spec.containers[*].name}')
   for container in $containers
   do
-    dockerfile=$(oc exec $pod -c $container -- find /root/buildinfo -name "Dockerfile-openshift3-logging*" || :)
+    dockerfile=$(oc -n $ns exec $pod -c $container -- find /root/buildinfo -name $pattern || :)
     if [ -n "$dockerfile" ]
     then
       echo Dockerfile info: $dockerfile > $env_file
-      oc exec $pod -c $container -- grep -o "\"build-date\"=\"[^[:blank:]]*\"" $dockerfile >> $env_file || echo ---- Unable to get build date
+      oc -n $ns exec $pod -c $container -- grep -o "\"build-date\"=\"[^[:blank:]]*\"" $dockerfile >> $env_file || echo ---- Unable to get build date
     fi
     echo -- Environment Variables >> $env_file
-    oc exec $pod -c $container -- env | sort >> $env_file
+    oc -n $ns exec $pod -c $container -- env | sort >> $env_file
   done
 }
 
 get_pod_logs() {
   local pod=$1
   local logs_folder=$2/logs
-  echo -- POD $1 Logs
+  local ns=${3:-$NAMESPACE}
+  echo ---- Logs for POD $1
   if [ ! -d "$logs_folder" ]
   then
     mkdir $logs_folder
   fi
-  local containers=$(oc get po $pod -o jsonpath='{.spec.containers[*].name}')
+  local containers=$(oc -n $ns get po $pod -o jsonpath='{.spec.containers[*].name}')
   for container in $containers
   do
-    oc logs $pod -c $container | nice xz > $logs_folder/$pod-$container.log.xz || oc logs $pod | nice xz > $logs_folder/$pod.log.xz || echo ---- Unable to get logs from pod $pod and container $container
+    echo ------ container: $container
+    oc -n $ns logs $pod -c $container | nice xz > $logs_folder/$pod-$container.log.xz || oc logs $pod | nice xz > $logs_folder/$pod.log.xz || echo ---- Unable to get logs from pod $pod and container $container
   done
-  if [ "$fluentd_folder" == "$2" ]
-  then
-    oc exec $1 logs | nice xz >> $logs_folder/$pod.log.xz
```

Comment:
Why this section is eliminated?  Both fluentd and rsyslog expect this command line to dump logs unless LOGGING_FILE_PATH is console.

Reply:
addressed here https://github.com/openshift/origin-aggregated-logging/pull/1715